main/src/library/Fixpoint/Solver.flix

```diff
@@ -45,8 +45,8 @@ namespace Fixpoint {
          lowerStmt |>
          interpret |>
          toAst) as & Pure;
-         let _ = notifyPostSolve(model);
-         model
```

Comment:
Why the change in indentation?

Reply:
Brings the let binding in line with the other let bindings.